faq/sql-faq.md

```diff
@@ -7,6 +7,28 @@ summary: Learn about the FAQs related to TiDB SQL.
 
 This document summarizes the FAQs related to SQL operations in TiDB.
 
+## Does TiDB support secondary key?
+
+Yes. If a column in TiDB meets all of the following conditions, it can work as a secondary key:
+
+- Not the primary key column
+- Have the [`NOT NULL` constraint](/constraints.md#not-null)
+- Have a [secondary index](/develop/dev-guide-create-secondary-indexes.md)
+
+## How does TiDB perform when executing DDL operations on a large table?
+
+DDL operations of TiDB on large tables are not an issue. TiDB supports online DDL operations, and these DDL operations do not block DML operations.
+
+For some DDL operations such as adding columns, deleting columns or dropping indexes, TiDB can perform these operations quickly.
+
+For some heavy DDL operations such as adding indexes, TiDB needs to backfill data, which takes a longer time (depending on the size of the table) and consumes additional resources. The impact on online traffic is tunable. TiDB can do the backfill with multiple threads, and you can tune the number of threads.
```

Comment:
It's better to replace `, and you can tune the number of threads` with `, the amount of thread is set by variables`

Reply:
addressed